ethereum/setprotocol_v2/insert_daily_component_prices.sql

```diff
@@ -240,6 +240,214 @@ with initial_components as (
   
 )
 SELECT count(*) + r INTO r from rows;
+-------------------------------------------------------------------------------------------------------------------------
+-- Step 3: Update prices for cETH using Compound-specific query
```

Comment:
it appears step 3 has duplicates on the values you implied in the 'ON CONFLICT' statement during inserts. please refer to this query to identify duplicates:
https://dune.com/queries/699027

i'm unable to load due to this failure

Reply:
Alright fixed, sorry about that, I was keeping two copies of the code, one using `dune_user_generated` and the other using `setprotocol_v2` and i didn't propagate all the changes over